bundle.Dockerfile

```diff
@@ -5,21 +5,16 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=cluster-logging
-LABEL operators.operatorframework.io.bundle.channels.v1=stable,stable-5.5
+LABEL operators.operatorframework.io.bundle.channels.v1=stable,stable-5.6
```

Comment:
should we maybe change the channel to "preview" or something we never use?

Reply:
Good point, I will do that.

Reply:
Done - the overlay name is used as the channel name.